shared/middleware/helpers/storage_apps.rb

```diff
@@ -29,7 +27,7 @@ def create(value, ip_address)
   end
 
   def delete(channel_id)
-    owner, id = storage_decrypt_channel_id_checked(channel_id)
+    owner, id = storage_decrypt_channel_id(channel_id)
     raise NotFound, "channel `#{channel_id}` not found in your storage" unless owner == @storage_id
```

Comment:
Outside the scope of this PR, but does this mean these NotFound errors will also log to honeybadger? Should we do something similar here?